File: CODE FILES/Backend/MyApiary/Resource.Api/Models/ApiaryUpdate.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class ApiaryUpdate
    {
        [Required]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Address { get; set; }

        [Required]
        public int UserId { get; set; }
    }
}

File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/ApiaryUpdate.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class ApiaryUpdate(
    @SerializedName("Id")
    val id: Int,
    @SerializedName("Name")
    val Name: String,
    @SerializedName("Address")
    val Address: String,
    @SerializedName("UserId")
    val userId: Int,
)
